VirtoCommerce.Storefront.Model/Cart/Services/ICartBuilder.cs

```diff
@@ -165,5 +165,10 @@ public interface ICartBuilder
         Task ValidateAsync();
 
         Task SaveAsync();
+
+
+        Task ChangeItemPriceAsync(string lineItemId, decimal price);
```

Comment:
Same as the commented above. Use `ChangeCartItemPrice` argument instead of multiple parameters.